csharp/test/Microsoft.ML.OnnxRuntime.Tests/Tensors/TensorTests.cs

```diff
@@ -160,7 +160,6 @@ public void ConstructFromDimensions(TensorConstructor tensorConstructor)
             Assert.Equal(tensorConstructor.IsReversedStride, tensor.IsReversedStride);
 
             //Assert.Throws<ArgumentNullException>("dimensions", () => tensorConstructor.CreateFromDimensions<int>(dimensions: null));
-            Assert.Throws<ArgumentException>("dimensions", () => tensorConstructor.CreateFromDimensions<int>(dimensions: new int[0]));
```

Comment:
This test is not valid anymore - in place of this - we have extensive tests covering multiple aspects of scalar tensors below.